amrit-docker-setup/start-all.sh

```
@@ -0,0 +1,48 @@
+#!/bin/bash
+set -euo pipefail
+
+echo "Starting infrastructure services (MySQL, Redis, MongoDB)..."
+docker compose -f docker-compose.infra.yml up -d
+
+# Load .env file if exists
+if [ -f .env ]; then
+  export $(grep -v '^#' .env | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)